test/gtest/ucs/test_vfs.cc

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#include <common/test.h>
+extern "C" {
+#include <ucs/vfs/sock/vfs_sock.h>
+}
+
+#include <sys/fcntl.h>
+#include <time.h>
+
+
+class test_vfs_sock : public ucs::test
+{
+protected:
+    virtual void init()
+    {
+        int ret = socketpair(AF_UNIX, SOCK_STREAM, 0, m_sockets);
+        if (ret < 0) {
```

Comment:
```
ASSERT_FALSE(ret < 0) << "failed to create socket pair: " << strerror(errno);
```
is it supported?

Reply:
it would fail assertion in https://github.com/yosefe/ucx/blob/9f5f8baaeb9746e52b26ddc1edb2dd054d7e080e/test/gtest/common/test.cc#L307